docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -435,6 +435,49 @@ incorporate the processes associated with crime reporting, the offense-trial
 pipeline, and an awareness of how the relationship between various social actors and
 the algorithm may impact the intended outcomes of a given model.
 
+
+The 4/5ths Rule: A Common Trap
+------------------------------
+
+In the literature around fairness in machine learning, one will often find
+the so-called "4/5ths rule" or "80% rule" used to assess whether a model
+(or mitigation technique) has produced a 'fair' result.
+Typically, the rule is implemented by using the demographic parity ratio introduced
```

Comment:
I would emphasize that this is the interpretation/translation computer scientists/fairness researchers have given to the rule.

This might be a good place to emphasize the difference between fairness and illegal discrimination.